src-tauri/tauri.conf.opencl.json

```diff
@@ -0,0 +1,113 @@
+{
+  "build": {
+    "distDir": "../dist/pwa",
+    "devPath": "http://localhost:3009",
+    "beforeDevCommand": "",
+    "beforeBuildCommand": "",
+    "features": ["opencl"]
+  },
+  "tauri": {
+    "systemTray": {
+      "iconPath": "icons/icon.png",
+      "iconAsTemplate": true
+    },
+    "bundle": {
+      "active": true,
+      "targets": "all",
+      "identifier": "subspace-desktop",
+      "icon": [
+        "icons/32x32.png",
+        "icons/128x128.png",
+        "icons/128x128@2x.png",
+        "icons/icon.icns",
+        "icons/icon.ico"
+      ],
+      "resources": [],
+      "externalBin": [],
+      "copyright": "",
+      "category": "DeveloperTool",
+      "shortDescription": "",
+      "longDescription": "",
+      "deb": {
+        "depends": ["ocl-icd-opencl-dev"]
```

Comment:
Dev package shouldn't be used in production dependencies, `ocl-icd-libopencl1` should be used instead, that is where `OpenCL.so` shared library comes from.